assets/sass/components/_buttons.scss

```diff
@@ -221,7 +221,7 @@ $buttonColor: get-color('brand', 'primary');
     text-decoration: underline;
     border: none;
     border-radius: 0;
-    cursor: inherit;
+    cursor: pointer;
```

Comment:
👎 Sorry! Can we add the pointer to _just_ those link buttons not all buttons

Reply:
But would we not need the `pointer` to all the buttons which looks like links.

Reply:
I think you're seeing the wrong classname here @davidrapson, this one **is** the link button class?

Reply:
Ah fair enough!